exec/java-exec/src/main/java/org/apache/drill/exec/ops/ContextInformation.java

```diff
@@ -29,13 +29,15 @@
   private final long queryStartTime;
   private final int rootFragmentTimeZone;
   private final String sessionId;
+  private final int hllAccuracy;
 
   public ContextInformation(final UserCredentials userCredentials, final QueryContextInformation queryContextInfo) {
     this.queryUser = userCredentials.getUserName();
     this.currentDefaultSchema = queryContextInfo.getDefaultSchemaName();
     this.queryStartTime = queryContextInfo.getQueryStartTime();
     this.rootFragmentTimeZone = queryContextInfo.getTimeZone();
     this.sessionId = queryContextInfo.getSessionId();
+    this.hllAccuracy = queryContextInfo.getHllAccuracy();
```

Comment:
Perhaps consider doing an @Inject some new context such as StatsContext into the UDF.